pkg/util/cgroups/errors.go

```diff
@@ -0,0 +1,72 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package cgroups
+
+import "fmt"
+
+type InvalidInputError struct {
```

Comment:
It can be good to document a bit more the different kind of errors, and when we should use it, and react to it.